src/Artsy/Relay/createRelaySSREnvironment.ts

```diff
@@ -76,63 +76,69 @@ export function createRelaySSREnvironment(config: Config = {}) {
+          headers: !!user
+            ? {
+                ...headers,
+                "X-USER-ID": user && user.id,
+                "X-ACCESS-TOKEN": user && user.accessToken,
+              }
+            : headers,
+        }),
+        relaySSRMiddleware.getMiddleware(),
+        cacheMiddleware({
+          size: 100, // max 100 requests
+          ttl: 900000, // 15 minutes
+          onInit: queryResponseCache => {
+            if (!isServer) {
+              hydrateCacheFromSSR(queryResponseCache)
             }
-          : headers,
-      }),
-      relaySSRMiddleware.getMiddleware(),
-      cacheMiddleware({
-        size: 100, // max 100 requests
-        ttl: 900000, // 15 minutes
-        onInit: queryResponseCache => {
-          if (!isServer) {
-            hydrateCacheFromSSR(queryResponseCache)
+          },
+        }),
+        next => async req => {
+          const res = await next(req)
+          const httpError = get(
+            res,
+            r => r.json.extensions.principalField.httpStatusCode
+          )
+          if (httpError) {
+            throw new HttpError(httpError)
+          } else {
+            return res
+          }
+        },
+        // TODO: This has been moved over from `Utils/metaphysics` but can eventually
+        // be replaced by error / retry middleware
+        next => async req => {
+          const response = await next(req)
+          if (
+            !checkStatus ||
+            (response.status >= 200 && response.status < 300)
+          ) {
+            return response
+          } else {
+            const error = new NetworkError(response.statusText)
+            error.response = response
+            throw error
           }
         },
-      }),
-      next => async req => {
-        const res = await next(req)
-        const httpError = get(
-          res,
-          r => r.json.extensions.principalField.httpStatusCode
-        )
-        if (httpError) {
-          throw new HttpError(httpError)
-        } else {
-          return res
-        }
-      },
-      // TODO: This has been moved over from `Utils/metaphysics` but can eventually
-      // be replaced by error / retry middleware
-      next => async req => {
-        const response = await next(req)
-        if (!checkStatus || (response.status >= 200 && response.status < 300)) {
-          return response
-        } else {
-          const error = new NetworkError(response.statusText)
-          error.response = response
-          throw error
-        }
-      },
-      loggingEnabled && loggerMiddleware(),
-      loggingEnabled && metaphysicsExtensionsLoggerMiddleware(),
-      loggingEnabled && errorMiddleware({ disableServerMiddlewareTip: true }),
-    ])
+        loggingEnabled && loggerMiddleware(),
+        loggingEnabled && metaphysicsExtensionsLoggerMiddleware(),
+        loggingEnabled && errorMiddleware({ disableServerMiddlewareTip: true }),
+      ],
+      { noThrow: true }
```

Comment:
We'll want a big bold `NOTE` / `TODO` pointing at the comment in the readme about 
> noThrow - EXPERIMENTAL (May be deprecated in the future) set true to not throw when an error response is given by the server, and to instead handle errors in your app code.

Reply:
Yea, I will clean this PR up and pop into the Relay slack to see about this.